Eat/Models/SearchQuery.swift

```diff
@@ -8,42 +8,42 @@
 
 import Foundation
 
-enum EatingTime {
+enum MealTime {
   case now
   case later(date: Date)
 }
 
-enum DietaryRestrictions {
+enum DietaryRestriction {
   case vegan, vegetarian, halal, none
 
-  var keyword: String? {
+  var keyword: String {
     switch self {
     case .vegan: return "vegan"
     case .vegetarian: return "vegetarian"
     case .halal: return "halal"
-    case .none: return nil
+    case .none: return ""
     }
   }
 }
 
 class SearchQuery {
   // Map
-  var latitude: Float = 49.2827 // Latitude of the restaurant
-  var longitude: Float = 123.1207 // Longitude of the restaurant.
+  var latitude: Double = 49.2827 // Latitude of the restaurant
```

Comment:
why do we have these values initialized here?

Reply:
Oh those are just default values. There's no initializers for the Search Query class, so when location changes, we actually go `searchQuery.latitude = 49.2`. In the case where they don't change it, the search query would still be valid?